readthedocs/vcs_support/backends/svn.py

```diff
@@ -39,7 +39,7 @@ def update(self):
         super(Backend, self).update()
         # For some reason `svn status` gives me retcode 0 in non-svn
         # directories that's why I use `svn info` here.
-        retcode = self.run('svn', 'info', record_as_success=True)[0]
+        retcode, _, _ = self.run('svn', 'info', record=False)
```

Comment:
Why we don't want to `record_as_success=True` here?

Reply:
The command is only used to check if the repo exists, so not relevant to the user. `as_success` makes the command to always return `0` as exit code, so no something we want.